src/commands/zombienet/init.ts

```diff
@@ -31,49 +36,31 @@ export class InitZombienet extends SwankyCommand<typeof InitZombienet> {
 
     const zombienetTemplatePath = getTemplates().zombienetTemplatesPath;
 
-    let provider = flags.provider;
-    if (provider === undefined) {
-      const answer = await inquirer.prompt([{
-        name: "provider",
-        type: "list",
-        choices: providerChoices,
-        message: "Select a provider to use",
-      }])
-      provider = answer.provider;
-    }
-
     const configPath = path.resolve(projectPath, "zombienet", "config")
-    // Copy templates
-    await spinner.runCommand(
-      () =>
-        copyTemplateFile(path.resolve(zombienetTemplatePath, provider!), path.resolve(configPath, provider!)),
-      "Copying template files"
-    );
+
+    if(!flags.binaries || flags.binaries.length < 2) {
```

Comment:
Polkadot binary should always be downloaded as default since it is the relay chain. So the default *zombienet.config.toml* should contain only relay chain config with the polkadot binary. Then additional and optional binaries can be downloaded such as `astar-collator` and `polkadot-parachain`. If so, the `buildZombienetConfigFromBinaries` helper should aggregate those specific parachains config to the *zombienet.config.toml* file.

Reply:
@ipapandinas Do you know where I can find `polkadot-parachain` binary?

Reply:
Here is the latest versions: https://github.com/paritytech/polkadot-sdk/releases/download/polkadot-v1.6.0/polkadot-parachain

Both `polkadot` & `polkadot-parachain` binaries are available in the [Polkadot SDK repository](https://github.com/paritytech/polkadot-sdk/releases)